### DISCUSSION TITLE
commit to master from different comuter

### DISCUSSION THREAD

**Discussion by MarkFuller1** [community/community#22605] [New to GitHub]
<p>Im not sure what they keywords for this would be.</p>
<p>I have an existing repo and i downloaded the code(onto a new machine) to edit but now i want to commit again. In retrospect i probably should have used the clone ability but hindsight is 20-20. What kind of setup do i need to connect my local directory in order to commit?</p>
<p>Or should I just delete the repo and create a new one?</p>

**Comment by Wabri** [comment, marked as answer]
<p>You have to clone the repo if you want to commit in a different pc</p>

**Comment by mpboom** [comment]
<p>Since you already downloaded it, then made changes you can also try cloning it to a different directory, then paste in your changes.</p>

**Comment by BaconBoi95** [comment]
<p><span class="hashtag">#AnnoyingSecurityFeatures</span></p>

**Comment by mpboom** [comment]
<p><a class="mention" href="/end3ryt">@end3ryt</a> this isn’t a security feature. In the original situation, the OP simply downloaded the repository ZIP without the .git folder. This way you just have the latest files, and no data on history, branches, tags, remotes, etc… Git will then just be unable to perform actions - it has nothing to do with security.</p>
